include/odp/api/spec/packet.h

```diff
@@ -1696,6 +1701,11 @@ odp_proto_l3_type_t odp_packet_l3_type(odp_packet_t pkt);
  *
  * Returns layer 4 protocol type. Initial type value is ODP_PROTO_L4_TYPE_NONE.
  *
+ * In addition to protocol types specified in ODP_PROTO_L4_TYPE_* defines,
+ * packet parser implementation may support parsing additional L4 protocol
+ * types. If protocol type is not recognized by parser, ODP_PROTO_L4_TYPE_NONE
+ * is returned.
```

Comment:
Same comment here: "... values from IANA list ..."

Reply:
Changed in V2.